pypesto/engine/mpi_pool.py

```diff
@@ -29,14 +31,19 @@ class MPIPoolEngine(Engine):
     def __init__(self):
         super().__init__()
 
-    def execute(self, tasks: List[Task]):
+    def execute(self, tasks: List[Task], progress_bar: bool = True):
         """Pickle tasks and distribute work to workers."""
```

Comment:
Would add a Parameter description here for progress_bar in the docstring. Same in the others.